Modules/@babylonjs/react-native/FontFace.ts

```diff
@@ -0,0 +1,35 @@
+import { Tools } from '@babylonjs/core';
+
+// Declare _native to get access to NativeCanvas.
+declare var _native: any;
+
+/**
+ * Partial Polyfill for FontFace Web API to wrap the NativeCanvas object.
+ */
+export class FontFace {
+  private source: string | ArrayBuffer | undefined;
```

Comment:
Maybe just `string` for now if that's all we are going to support?

Reply:
I'm assigning the source to undefined after we load the font so that we don't hold two copies of it in memory